.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,22 @@
 <!--
-# nf-core/vcftomaf pull request
+# qbic-pipelines/vcftomaf pull request
 
-Many thanks for contributing to nf-core/vcftomaf!
+Many thanks for contributing to qbic-pipelines/vcftomaf!
 
 Please fill in the appropriate checklist below (delete whatever is not relevant).
 These are the most common things requested on pull requests (PRs).
 
 Remember that PRs should be made against the dev branch, unless you're preparing a pipeline release.
 
-Learn more about contributing: [CONTRIBUTING.md](https://github.com/nf-core/vcftomaf/tree/master/.github/CONTRIBUTING.md)
+Learn more about contributing: [CONTRIBUTING.md](https://github.com/qbic-pipelines/vcftomaf/tree/master/.github/CONTRIBUTING.md)
 -->
 
 ## PR checklist
 
 - [ ] This comment contains a description of changes (with reason).
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
-- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/vcftomaf/tree/master/.github/CONTRIBUTING.md)
-- [ ] If necessary, also make a PR on the nf-core/vcftomaf _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
+- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/qbic-pipelines/vcftomaf/tree/master/.github/CONTRIBUTING.md)
+- [ ] If necessary, also make a PR on the qbic-pipelines/vcftomaf _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
```

Comment:
here possibly as well? depending on which testdata you  are using. In any case it is not the vcf2maf branch